exercises/concept/cache/cache_struct.go

```diff
@@ -0,0 +1,31 @@
+package cache
```

Comment:
If you follow the suggestion mentioned above with the interface, the cache implementation can be part of the standard test file. If you want to keep it as a separate file, you can list it in the config under the "editor" key so it shows up when solving the exercise but cannot be edited on the website.